test/cpuinfo_s390x_test.cc

```diff
@@ -0,0 +1,86 @@
+// Copyright 2018 IBM.
```

Comment:
minor comment: could you make the copyright comment consistent? I see that in above files `Copyright 2022 IBM`